mne/source_estimate.py

```diff
@@ -2615,7 +2615,12 @@ def save_stc_as_volume(fname, stc, src, dest='mri', mri_resolution=False):
     shape3d = (shape[2], shape[1], shape[0])
     shape = (n_times, shape[2], shape[1], shape[0])
     vol = np.zeros(shape)
-    mask3d = src[0]['inuse'].reshape(shape3d).astype(np.bool)
+    if np.array_equal(stc.vertices, src[0]['vertno']):
```

Comment:
can you add a comment about why this check is necessary / what the branches imply?